kolibri/auth/models.py

```diff
@@ -761,13 +762,14 @@ class Membership(AbstractFacilityDataModel):
 
     permissions = (
         IsOwn(read_only=True) |  # users can read their own Memberships
-        RoleBasedPermissions(  # Memberships can be read and written by admins, and read by coaches, for the member user
+        RoleBasedPermissions(  # Memberships can be read and written by admins for the member user
             target_field="user",
-            can_be_created_by=(role_kinds.ADMIN,),
```

Comment:
These arguments need to be tuples. If you delete the inner comma, it's no longer a tuple. Same with the ones below.